libs/api/prisma/migrations/20240909161100_delete_preferred_name/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `preferredName` on the `users` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "users" DROP COLUMN "preferredName";
```

Comment:
I will double-check with tsri.

Reply:
This story was about Tsri not wanting this